ecosystem/indexer-grpc/indexer-grpc-post-processor/src/main.rs

```diff
@@ -88,6 +95,23 @@ impl RunnableConfig for IndexerGrpcPostProcessorConfig {
             }));
         }
 
+        if let Some(config) = &self.processor_status_checker_config {
+            tasks.push(tokio::spawn({
+                let config = config.clone();
+                async move {
+                    let checker =
+                        ProcessorStatusChecker::new(config.hasura_rest_api_endpoint.clone());
+                    info!("Starting ProcessorStatusChecker");
```

Comment:
is it ok to expose `hasura_rest_api_endpoint` in the `info`?

Reply:
yes I think it's fine since it's a publicly available endpoint.